api/game/gameRouter.js

```diff
@@ -373,11 +373,11 @@ router.get('/faceoffs', authRequired, (req, res) => {
   crudOperationsManager.getAll(res, Game.getAllFaceOffs, 'Faceoffs')
 });
 
-router.get('/faceoffs/:id', authRequired, (req, res) =>{
-  const { id } = req.params;
+// router.get('/faceoffs/:id', authRequired, (req, res) =>{
+//   const { id } = req.params;
 
-  crudOperationsManager.getById(res, Game.getFaceOffByID, 'Faceoffs', id);
-})
+//   crudOperationsManager.getById(res, Game.getFaceOffByID, 'Faceoffs', id);
+// })
 
```

Comment:
this was definitely a tricky bug, but I think I'll always remember the lesson that dynamic routes in the backend will break everything below it

Reply:
Yes! Something I also learned in labs, if only I could commit it to memory!